### DISCUSSION TITLE
Images are greyed when in Dark theme.

### DISCUSSION THREAD

**Discussion by warcaninos** [TriliumNext/Trilium#4149] [Q&A] [answered]
Hi like the title say the images in dark theme are greyer than in light theme and i didn't see any custom theme tackle this issue, is it even possile with CSS if so can someone orient me to the right direction.
Thanks.
Light theme
![image](https://github.com/zadam/trilium/assets/141662338/4150abdc-c17a-46d4-aad6-3e3980133ec1)
Dark theme
![image](https://github.com/zadam/trilium/assets/141662338/b71bf188-fbbe-4a4a-9135-8dc81e844305)

**Comment by zadam** [comment]
Hello, I can't reproduce it with built-in themes. Are you sure you're not using some custom theme?

Also, the first screenshow doesn't look like it comes from the "light" theme ...

**Reply by warcanin** [reply]
Indeed i should have disabled the steel blue theme.

Here is the original picture.
![image](https://github.com/zadam/trilium/assets/57159438/bbd0afee-2c2f-4b81-a877-6b4a9b66198d)

Here is the picture with the light theme.
![image](https://github.com/zadam/trilium/assets/57159438/524b86f2-bf8e-4239-ad2c-3bccc90b9c47)

Here is the picture with the dark theme.
![image](https://github.com/zadam/trilium/assets/57159438/68b8418c-dcf6-42e9-a3fb-547558553cfd)

**Reply by zadam** [reply, marked as answer]
Oh, this is in the canvas type note. Then yes, that's what happens. I believe it's because the excalidraw (library powering the canvas type) uses a simple/primitive way to create a dark theme using CSS filters which then produce this darkening. So it's up to excalidraw team to implement proper themes ...

Some discussion in https://github.com/excalidraw/excalidraw/issues/4616 and https://github.com/excalidraw/excalidraw/issues/3531

**Reply by warcanin** [reply]
Sorry i forgot to thank you this is my second account.
